great_expectations/rule_based_profiler/config/base.py

```diff
@@ -233,6 +249,11 @@ def __init__(
         if meta is not None:
             self.meta = meta
 
+        if validation_parameter_builder_configs:
+            self.validation_parameter_builder_configs = (
+                validation_parameter_builder_configs
+            )
+
```

Comment:
Same notes here

Reply:
@cdkini I think that this is totally fine here -- we do not want empty (`None`) elements having to watch for in all tests.  We use this pattern in other places.  If we did not, the very `great_expectations.yml` would be full of keys with no value.